jenkins/run_upload_es.py

```diff
@@ -67,11 +67,10 @@ def get_custodian_policies(type: str = None):
 
 
 es_index = 'cost-explorer-perf'
-metric_type = 'BlendedCost'
```

Comment:
granularity instead of report_type [put all the option here Daily, Monthly, Hourly]
cost_metric parameter instead of hard coded 'BlendedCost'
You should get granularity and cost_metric by parameters from environment variable